.gitignore

```diff
@@ -79,7 +79,6 @@ shed_tool_data_table_conf.xml
 job_conf.xml
 data_manager_conf.xml
 shed_data_manager_conf.xml
-object_store_conf.xml
```

Comment:
Why this change?

Reply:
In order to test irods, I need to override the objectstore to be irods.  I need this file checked in (which has irods details).I guess I could re-name the file, if this must remain in .gitignore.

Reply:
You can use `git add --force PATH` to check in a file that would be normally ignored. After that, git will take into account modifications to it in `git status` and so on.

Reply:
Correct. I think I will undo the change to .gitignore file and rename the config file to irods_object_store_conf.xml instead.